network.go

```diff
@@ -13,34 +12,25 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-const fsGetNetInterfacesTimeout = time.Second * 10
+const netGetCountersTimeout = time.Second * 10
 
 type netWatcher struct {
 	cagent           *Cagent
 	lastIOCounters   []utilnet.IOCountersStat
 	lastIOCountersAt *time.Time
 
-	ExcludedInterfaceCache map[string]bool
+	netInterfaceExcludeRegexCompiled []*regexp.Regexp
+	constantlyExcludedInterfaceCache map[string]bool
 }
 
 func (ca *Cagent) NetWatcher() *netWatcher {
-	return &netWatcher{cagent: ca, ExcludedInterfaceCache: map[string]bool{}}
+	return &netWatcher{cagent: ca, constantlyExcludedInterfaceCache: map[string]bool{}}
 }
 
-func (nw *netWatcher) Results() (MeasurementsMap, error) {
-	results := MeasurementsMap{}
-
-	var errs []string
-	ctx, cancel := context.WithTimeout(context.Background(), fsGetNetInterfacesTimeout)
-	defer cancel()
-
-	interfaces, err := utilnet.Interfaces()
-
-	if err != nil {
-		log.Errorf("[NET] Failed to read interfaces: %s", err.Error())
-		errs = append(errs, err.Error())
+func (nw *netWatcher) InterfaceExcludeRegexCompiled() []*regexp.Regexp {
```

Comment:
Please add a (GoDoc compatible) summary what the function does.

Reply:
done. Is it clear enough?